src/TabbedArea.js

```diff
@@ -108,12 +108,14 @@ const TabbedArea = React.createClass({
   },
 
   renderTab(child) {
-    let key = child.props.eventKey;
+    let {eventKey, className, tab, ...other} = child.props;
```

Comment:
The variable 'child' is not accurate.  It is the existing element being
rendered.  In the 'render' function, it calls:

ValidComponentChildren.map(this.props.children, renderTabIfSet, this)

I think you are right and that I should remove this.  It would copy the
TabPane properties to the corresponding Nav header.

On Tue, Apr 14, 2015 at 12:53 PM, Alexander Shemetovsky <
notifications@github.com> wrote:

> In src/TabbedArea.js
> <https://github.com/react-bootstrap/react-bootstrap/pull/517#discussion_r28363804>
> :
>
> > @@ -108,12 +108,14 @@ const TabbedArea = React.createClass({
> >    },
> >
> >    renderTab(child) {
> > -    let key = child.props.eventKey;
> > +    let {eventKey, className, tab, ...other} = child.props;
>
> For what reason all ...other props could be applied from child to parent ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/react-bootstrap/react-bootstrap/pull/517/files#r28363804>
> .
>